.changeset/rich-worms-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte': patch
+---
+
+fix: ensure spread attribute events are attached synchronously
```

Comment:
let's use active voice for changesets, otherwise everything is 'ensure this' or 'ensure that'

```suggestion
fix: attach spread attribute events synchronously
```